### PR TITLE
Destination Tile

### PR DESCRIPTION
**Pull request by jackcarlisle**
## ready for review

* adds the destination tile with dummy data

### CODE REVIEW

File: src/assets/map.svg [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	 viewBox="0 0 257.853 257.853" style="enable-background:new 0 0 257.853 257.853;" xml:space="preserve">
+<path d="M6.02,103.222V251l11.46,6.37l64.541-40.124l91.35,40.608l7.006-0.484l67.917-42.222l3.54-6.37V61l-11.46-6.37
+	l-64.542,40.124L151.31,83.854c2.407-6.428,3.605-12.365,3.605-17.854c0-36.393-29.607-66-66-66s-66,29.607-66,66
+	c0,6.047,1.453,12.637,4.377,19.829L9.56,96.853L6.02,103.222z M88.915,15c28.166,0,51,22.833,51,51s-51,76-51,76s-51-47.834-51-76
+	S60.748,15,88.915,15z M21.02,107.391l13.04-8.106c4.523,7.653,10.438,15.905,17.777,24.815
+	c13.211,16.039,26.267,28.327,26.816,28.842l10.262,9.624l10.262-9.624c0.549-0.515,13.605-12.803,26.816-28.842
+	c7.944-9.645,14.221-18.519,18.869-26.697l28.51,12.673l7.006-0.484l56.457-35.098v130.116l-61.001,37.923l-91.35-40.608
+	l-7.006,0.484L21.02,237.506V107.391z M118.915,65c0-16.542-13.458-30-30-30s-30,13.458-30,30s13.458,30,30,30
+	S118.915,81.542,118.915,65z M73.915,65c0-8.271,6.729-15,15-15s15,6.729,15,15s-6.729,15-15,15S73.915,73.271,73.915,65z"/>
+<g>
```
Comment (nelsonic) [inline src/assets/map.svg]: fell free to remove these excess `<g></g>` tags...

File: src/assets/video-camera.svg [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+			c-7.024,0-12.738,5.715-12.738,12.738c0,3.867,1.489,7.331,4.219,9.669C3.055,37.481,0,40.848,0,44.866z M74.121,39.577
+			C74.723,39.76,75,40.387,75,40.976v25.892c0,0.59-0.277,1.218-0.877,1.401L58,60.125V47.678L74.121,39.577z M26.742,32.176
+			c1.039,1.464,2.307,2.813,3.75,3.813h-6.637C25.042,34.989,26.024,33.625,26.742,32.176z M27.63,22.843
+			c0-7.075,5.755-12.831,12.829-12.831c7.075,0,12.83,5.756,12.83,12.831c0,7.073-5.755,12.828-12.83,12.828
+			c-6.016,0-11.063-4.168-12.443-9.764c-0.056-0.693-0.17-1.368-0.333-2.025C27.655,23.538,27.63,23.193,27.63,22.843z
+			 M15.33,18.194c3.97,0,7.324,2.664,8.384,6.296c0.067,0.689,0.185,1.363,0.333,2.026c0.006,0.139,0.021,0.275,0.021,0.416
+			c0,4.818-3.92,8.738-8.738,8.738c-4.819,0-8.739-3.92-8.739-8.738S10.511,18.194,15.33,18.194z M54,64.493
+			c0,2.483-2.013,4.496-4.496,4.496H8.496C6.013,68.989,4,66.976,4,64.493V44.485c0-2.483,2.013-4.496,4.496-4.496h41.008
+			c2.483,0,4.496,2.013,4.496,4.496V64.493z"/>
+	</g>
```
Comment (nelsonic) [inline src/assets/video-camera.svg]: same again.

File: lib/destination-tile/test/index.test.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import DestinationTile from '../';
+
+const props = {
+  destination: {
+    sections: [
```
Comment (nelsonic) [inline lib/destination-tile/test/index.test.js]: I would *consider* having a better content fixture ... but this is ok for now.

**Comment by nelsonic** [comment]
@jackcarlisle component looks good. 
if you have time, please tidy up the SVGs 
then happy to merge. 👍

**Comment by jackcarlisle** [comment]
@nelsonic should be ready to go now

**Comment by nelsonic** [comment]
@jackcarlisle also, consider updating the PR description (from "_not ready_") *before* assigning for review... 😉

**Comment by nelsonic** [comment]
Cool. merging. 👍

**Pull request merged**